bubble_sort/bubblesort.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpry_ry99g/bubble_sort/bubblesort.py
+++ b/tmp/tmpry_ry99g/bubble_sort/bubblesort.py
@@ -2,7 +2,7 @@
 Bubble sort
 
 It is the most common sort algorithm.
-It simply finds the first element and place at 
+It simply finds the first element and place at
 the first by comparing all elements.
 
 you should use the sort built in python 
```